### PR TITLE
👷  Updating Versions

### PR DESCRIPTION
**Pull request by aridutilh**
Basically just changing how we do the versions pages - now the main versions page has the headers allowing people to stay on the site, and the individual versions pages have target="_blank" so they're still technically on our site, but people don't have to click the back button, they can just close out and go back to our site where we're still able to control the header.

### CODE REVIEW

**Comment by MattA-Official** [comment]
This would conflict with #1 I would recommend using [`nuxt`](https://github.com/PhotographyLounge/site/tree/nuxt) as your base branch

**Review by MattA-Official — COMMENTED** [review]
This would conflict with #1 I would recommend using [`nuxt`](https://github.com/PhotographyLounge/site/tree/nuxt) as your base branch

**Comment by aridutilh** [comment]
> This would conflict with #1 I would recommend using [`nuxt`](https://github.com/PhotographyLounge/site/tree/nuxt) as your base branch

base branch as in I submit my code into your pr or base branch as is I make a pr to your pr

File: discord.photography/Portfolios.html [hunk 1]
```diff
@@ -21,14 +21,13 @@
...
-			<li><a href="Portfolios.html">Portfolios&nbsp;</a></li>
-			
-			<li><a href="Staff.html">Staff</a></li>
+			<a href="index.html">Home &nbsp;</a>
+	
+			<a href="Versions.html">Versions &nbsp;</a>
+	
+			<a href="Portfolios.html">Portfolios&nbsp;</a>
+	
+			<a href="Staff.html">Staff</a>
```
Comment (lostdesign) [inline discord.photography/Portfolios.html:30]: We should not use `&nbsp;` for a navigation that needs spacing inbetween. Rather should use margin between the elements or use flex on the container with `space-between`

File: discord.photography/Portfolios.html [hunk 2]
```diff
@@ -21,14 +21,13 @@
 
 	<div class="main">
 		<h4>
```
Comment (lostdesign) [inline discord.photography/Portfolios.html:23]: Its not really good wrapping all links in a single h4 - accessibility wise its also not a h4, maybe h1 if at all and then style it as h4. Yet would rather not use it at all and just style the hrefs. Since this is out of scope of this PR, we may need to change that in a different PR

File: discord.photography/Portfolios.html [hunk 3]
```diff
@@ -21,14 +21,13 @@
...
-
-			<li><a href="index.html">Home &nbsp;</a></li>
-
-			<li><a href="Updates/Versions.html">Updates &nbsp;</a></li>
-			
-			<li><a href="Portfolios.html">Portfolios&nbsp;</a></li>
-			
-			<li><a href="Staff.html">Staff</a></li>
+			<a href="index.html">Home &nbsp;</a>
```
Comment (lostdesign) [inline discord.photography/Portfolios.html:24]: Actually, why are we using HTML when we are using nuxt?

**Review by lostdesign — CHANGES_REQUESTED** [review]
We should use `nuxt-link` exclusively for internal routing inside the nuxt app.

**Comment by lostdesign** [comment]
Closing this since it touches files which are soon to be removed.
